Encode_Club_AI_and_GPT_Bootcamp_Q3_Homework_for_week_1.ipynb

```diff
@@ -105,25 +113,25 @@
       "outputs": [],
       "source": [
         "# Create a new client instance\n",
-        "client = OpenAI(api_key=\"XXXXXXXXXXXXXXXXXXXXXXXXXXX\")"
+        "client = OpenAI(api_key=userdata.get('OPEN_AI_API_KEY'))"
```

Comment:
Nice, you source the API key from Google Colab user data?
I'll try to check my Notebook, too